test/CrateTest/PDO/PDOStatementTest.php

```diff
@@ -105,8 +106,8 @@ private function getPopulatedCollection()
      */
     public function testInstantiation()
     {
-        $this->assertInstanceOf('Crate\PDO\PDOStatement', $this->statement);
-        $this->assertInstanceOf('PDOStatement', $this->statement);
+        $this->assertInstanceOf(PDOStatement::class, $this->statement);
+        $this->assertInstanceOf(\PDOStatement::class, $this->statement);
```

Comment:
What's the difference between those 2 assertions?

Reply:
@matriv One is the native PDOStatement and one is our wrapper around it